src/assets/wise5/authoringTool/create-branch-paths/create-branch-paths.component.ts

```diff
@@ -117,8 +122,11 @@ export class CreateBranchPathsComponent {
     }
   }
 
-  protected initializePathValues(): void {
-    if (this.criteria === this.CHOICE_CHOSEN_VALUE) {
+  protected initializeChoiceChosenOptions(): void {
+    if (
+      this.criteria === this.CHOICE_CHOSEN_VALUE &&
+      this.getComponent()?.type === 'MultipleChoice'
+    ) {
```

Comment:
No need to check criteria again?
```suggestion
    if (this.getComponent()?.type === 'MultipleChoice') {
```